corehq/apps/locations/_design/views/users_by_location_id/map.js

```diff
@@ -0,0 +1,5 @@
+function(doc) {
```

Comment:
i think we could use elastic for this - though if this is called in an OTA restore workflow then we should just bite the bullet and add a couch view.